src/System.ComponentModel.TypeConverter/ref/System.ComponentModel.TypeConverter.cs

```diff
@@ -8,6 +8,7 @@
 using System.Runtime.Serialization;
 
 [assembly: System.Runtime.CompilerServices.TypeForwardedTo(typeof(System.ComponentModel.Component))]
+[assembly: System.Runtime.CompilerServices.TypeForwardedTo(typeof(System.ComponentModel.ISupportInitialize))]
```

Comment:
Hmmm. there must be something else it is consuming still building against ISupportInitialize from TypeConverter. At any rate it isn't really an issue to have the type-forward here so you can keep it.